docker/airflow/1.10.5/Dockerfile

```diff
@@ -85,13 +86,13 @@ RUN apk update \
 		py3-typed-ast@edge-testing \
 		python3 \
 		tini \
-	&& update-ca-certificates \
-	&& cp /usr/share/zoneinfo/UTC /etc/localtime \
+	&& update-ca-certificates
+RUN cp /usr/share/zoneinfo/UTC /etc/localtime \
 	&& pip3 install --no-cache-dir --upgrade pip==19.2.1 \
 	&& pip3 install --no-cache-dir --upgrade setuptools==41.0.1 \
 	&& pip3 install --no-cache-dir "${AIRFLOW_MODULE}" \
-	&& pip3 install --no-cache-dir "https://github.com/astronomer/astronomer-fab-securitymanager/releases/download/v1.2.1/astronomer_fab_security_manager-1.2.1-py3-none-any.whl" \
-	&& cd /usr/lib/python3.7/site-packages/airflow/www_rbac \
+	&& pip3 install --no-cache-dir "https://github.com/astronomer/astronomer-fab-securitymanager/releases/download/v1.2.1/astronomer_fab_security_manager-1.2.1-py3-none-any.whl"
+RUN cd /usr/lib/python3.7/site-packages/airflow/www_rbac \
 	&& npm install \
 	&& npm run build \
 	&& rm -rf node_modules \
```

Comment:
If we split this command out in to a different layer then deleting the files doesn't save us any space on the pull.

Reply:
Well - some from this, but the next line (`apk del .build-deps py3-numpy-dev`) doesn't save much space.

Reply:
Yeah that's true... so I guess the question is does the increased-space counteract the better distribution of layers?

Reply:
Another option could be a multi-stage build, but not sure if its worth that effort or not.